nyx_client/nyx_client/client.py

```diff
@@ -602,6 +606,7 @@ def create_data(
             circles: A list of circles to add share the data with
             custom_metadata: Additional metadata properties to decorate the data with. Note that nyx-internal properties
                 are not allowed.
+            connection_id: the id of a connection to use (id from :class`Connection`)
```

Comment:
Generated docs don't contain a link. Either `:class:`.Connection` or absolute fqdn does work

Reply:
Still needs `:` after `class` to render link